CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing Guide
+
+First of all, thank you for joinjoining us to make this project better. Here are some guidelines to help you contribute to this project. Hoping you feel they are useful.
```

Comment:
1. `thank you for joinjoining` - Should be "joining"
1. `Hoping you feel they are useful.` - I think you could do without this. Either way, it should be "We hope that you find them useful." (It sounds better)